sdk/servicebus/service-bus/samples-v1/javascript/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "azure-service-bus-samples-ts",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Azure Service Bus client library samples for TypeScript",
+  "engine": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "prebuild": "rimraf dist/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git"
+  },
+  "keywords": [
+    "Azure",
+    "Service Bus",
+    "Node.js",
+    "TypeScript"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "sideEffects": false,
+  "dependencies": {
+    "@azure/identity": "^1.0.2",
+    "@azure/service-bus": "latest",
```

Comment:
probably want to pin a version here. Once the next GA ships, this will start to resolve strangely.

Reply:
Agree, I'll pin it to 1.1.6.